Bake Default Settings

robertlong13 opened the pull request:
Adds default settings so that a fresh install is (hopefully) fully ready to go.

Lachie, give that custom.config.xml a compare with your stuff. I tried removing as many lines as I could though, so expect big diffs.

LachlanConn commented on custom.config.xml:
```diff
@@ -0,0 +1,163 @@
+	<guided_alt>100</guided_alt>
+	<HUD_showicons>True</HUD_showicons>
+	<hud1_useritem_DistToHome>DistToHome: </hud1_useritem_DistToHome>
+	<hud1_useritem_sonarrange>Lidar: </hud1_useritem_sonarrange>
```
kind of hard to review but I did find a difference here. I have 10 preceding spaces to have the message in the right location

LachlanConn commented on custom.config.xml:
```diff
@@ -0,0 +1,163 @@
+	<TXT_loiterrad>250</TXT_loiterrad>
+	<TXT_WPRad>75.00</TXT_WPRad>
+	<UDP_BAUD>115200</UDP_BAUD>
+	<UDP_port>14552</UDP_port>
```
should be 14550?

robertlong13 commented:
I removed warning since we are not releasing Cx7.1 this month, but don't want to hold off the Planner changes, which can otherwise go in.

Also, I made some other changes to the warnings.
- Removing altitude error warning. This would go off any time you change your target altitude.
- Removing the terrain clearance. This will be constantly going off every takeoff and landing, which _maybe_ we could tolerate. I'm pretty sure it will go off randomly when you fly powerlines every time you pass over any part of the tower.
- Changed the GPS warnings to repeat every 60 seconds instead of every 10.
- Changed the temperature warnings to repeat every 30 seconds instead of every 10.
